lemur/tests/factories.py

```diff
@@ -43,6 +53,7 @@ class CertificateFactory(BaseFactory):
     description = FuzzyText(length=128)
     active = True
     date_created = FuzzyDate(date(2016, 1, 1), date(2020, 1, 1))
+    rotation_policy = SubFactory(RotationPolicyFactory)
```

Comment:
It would be nicer to re-use the same 'default' policy every time, but I don't know how to make that happen.